storio-common/src/test/java/com/pushtorefresh/storio/operations/internal/OnSubscribeExecuteAsBlockingCompletableTest.java

```diff
@@ -0,0 +1,64 @@
+package com.pushtorefresh.storio.operations.internal;
+
+import com.pushtorefresh.storio.StorIOException;
+import com.pushtorefresh.storio.operations.PreparedWriteOperation;
+
+import org.junit.Test;
+
+import rx.Completable;
+import rx.observers.TestSubscriber;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+public class OnSubscribeExecuteAsBlockingCompletableTest {
+
+    @SuppressWarnings("ResourceType")
+    @Test
+    public void shouldExecuteAsBlockingAfterSubscription() {
+        final PreparedWriteOperation preparedOperation = mock(PreparedWriteOperation.class);
+
+        TestSubscriber testSubscriber = new TestSubscriber();
+
+        verifyZeroInteractions(preparedOperation);
```

Comment:
Happens to everybody :)

On Fri, 18 Mar 2016, 00:07 Ilya, <notifications@github.com> wrote:

> In
> storio-common/src/test/java/com/pushtorefresh/storio/operations/internal/OnSubscribeExecuteAsBlockingCompletableTest.java
> <https://github.com/pushtorefresh/storio/pull/629#discussion_r56577872>:
>
> > +import static org.mockito.Mockito.mock;
> > +import static org.mockito.Mockito.times;
> > +import static org.mockito.Mockito.verify;
> > +import static org.mockito.Mockito.verifyZeroInteractions;
> > +import static org.mockito.Mockito.when;
> > +
> > +public class OnSubscribeExecuteAsBlockingCompletableTest {
> > +
> > +    @SuppressWarnings("ResourceType")
> > +    @Test
> > +    public void shouldExecuteAsBlockingAfterSubscription() {
> > +        final PreparedWriteOperation preparedOperation = mock(PreparedWriteOperation.class);
> > +
> > +        TestSubscriber testSubscriber = new TestSubscriber();
> > +
> > +        verifyZeroInteractions(preparedOperation);
>
> Right, not sure what I was thinking about
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/629/files/a2d8e90a41cffedb7ca712ab109f4d27eb706e40#r56577872>
>
-- 

@artem_zin